pandas/core/common.py

```diff
@@ -700,3 +700,53 @@ def deprecate_numeric_only_default(cls: type, name: str, deprecate_none: bool =
     )
 
     warnings.warn(msg, FutureWarning, stacklevel=find_stack_level())
+
+
+def resolve_na_sentinel(
```

Comment:
can you put this somewhere else, closer to where its used

Reply:
Moved to algorithms.